include/igraph_adjlist.h

```diff
@@ -188,8 +187,7 @@ DECLDIR int igraph_adjedgelist_init(const igraph_t *graph,
                                     igraph_inclist_t *il,
                                     igraph_neimode_t mode);
 DECLDIR void igraph_adjedgelist_destroy(igraph_inclist_t *il);
-DECLDIR int igraph_adjedgelist_remove_duplicate(const igraph_t *graph,
-        igraph_inclist_t *il);
+DECLDIR int igraph_adjedgelist_remove_duplicate(igraph_inclist_t *il);
```

Comment:
If we deprecate `adjedgelist_*` functions (we should), we should not change them anymore.

Reply:
Yes, that's true, although it doesn't hurt to modify them one last time here. :) I leave it up to Vincent to decide whether he tackles the removal in this PR as well.

Reply:
As said in other comments, let's do this in a separate PR.